.github/workflows/release-build.yaml

```diff
@@ -339,7 +339,7 @@ jobs:
           - nuclei
           - screenshooter
           - semgrep
-          - ssh-scan
```

Comment:
ssh-scan should still be build until we completely remove it, same for the telemetry :)